frontends/infinite-corridor/src/pages/FieldPage.tsx

```diff
@@ -1,21 +1,73 @@
-import React from "react"
-import { useParams } from "react-router"
-import { Card } from "ol-util"
+import React, { useCallback } from "react"
+import { useParams, useLocation } from "react-router"
+import Tab from "@mui/material/Tab"
+import TabContext from "@mui/lab/TabContext"
+import TabList from "@mui/lab/TabList"
+import TabPanel from "@mui/lab/TabPanel"
+import Container from "@mui/material/Container"
+import Divider from "@mui/material/Divider"
+import Grid from "@mui/material/Grid"
+import { useFieldDetails } from "../api/fields"
+import { Link } from "react-router-dom"
+import FieldPageSkeleton from "./FieldPageSkeleton"
 
 type RouteParams = {
   name: string
 }
 
+const keyFromHash = (hash: string) => {
+  const keys = ["home", "about"]
+  const match = keys.find(key => `#${key}` === hash)
+  return match ?? "home"
+}
+
 const FieldPage: React.FC = () => {
   const { name } = useParams<RouteParams>()
+  const { hash } = useLocation()
+
+  const [value, setValue] = React.useState(keyFromHash(hash))
+  const field = useFieldDetails(name)
+  const handleChange = useCallback(
+    (event: React.SyntheticEvent, newValue: string) => {
+      setValue(newValue)
+    },
+    []
+  )
 
   return (
-    <div className="page-content">
-      <Card>
-        <h2>{name}</h2>
-        Welcome to the field page for: <code>{name}</code>!
-      </Card>
-    </div>
+    <FieldPageSkeleton name={name}>
+      <TabContext value={value}>
+        <Container>
```

Comment:
We do breakpoints ourselves in open-discussions.

But breakpoints are complicated...which I came to appreciate (even more) after looking at https://material.io/design/layout/responsive-layout-grid.html#breakpoints

Let's let MUI handle it. We *can* customize the breakpoint stuff if we want to. (e.g., to make it more consistent with open-discussions).